spec/features/posts_spec.rb

```diff
@@ -1,9 +1,13 @@
 require 'rails_helper'
 
 RSpec.feature "Posts", type: :feature do
-  scenario "creates a new blog post" do
-    visit new_post_path
-    sleep(5)
-    expect(page).to have_content("New Post")
+  scenario "displays published posts ONLY" do
+
+    post = Post.create(title: "Test Title", body: "Test Body", published_at: Date.tomorrow)
+    expect {
+      visit root_path
+      expect(page).to have_content "Posts"
+      expect(page).to_not have_content "Test Title"
+    }.to change(Post.all, :count).by(0)
```

Comment:
You don't need to check this? I feel like viewing a post should never change Post count. You can just change your `scenario` to an `it`.